### PR TITLE
Feature/54/create login popup

### PR DESCRIPTION
**Pull request by tarasSanotskyi**
Create login popup
Create useForm hook

image
image
image

### CODE REVIEW

File: src/containers/modal/Modal.js [hunk 1]
```diff
@@ -17,7 +17,11 @@ const Modal = ({ content, closeModal }) => {
   }
 
   return (
-    <Dialog maxWidth="xl" onClose={ onClose } open>
+    <Dialog
+      // fullScreen
```
Comment (maarekua) [inline src/containers/modal/Modal.js:21]: // @TODO fullscreen

Comment (tarasSanotskyi) [reply src/containers/modal/Modal.js]: fixed

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/components/google-login/GoogleLogin.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { useTranslation } from 'react-i18next'
+import { Box, Typography, Button } from '@mui/material'
+
+import style from './google-login.style'
+import google from '~/assets/img/login-dialog/google.svg'
+
+
+const GoogleLogin = () => {
```
[the remark this answers is not in the document]
Comment (tarasSanotskyi) [reply src/components/google-login/GoogleLogin.js]: fixed

File: src/components/login-dialog/LoginDialog.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import GoogleLogin from '../google-login/GoogleLogin'
+import LoginForm from '../login-form/LoginForm'
+import useForm from '~/hooks/use-form'
+import useConfirm from '~/hooks/use-confirm'
+import usePrompt from '~/hooks/use-prompt'
+import { email, password } from '~/constants/validation/login'
+import login from '~/assets/img/login-dialog/login.svg'
+import style from './loginDialog.style'
+
+const LoginDialog = () => {
```
[the remark this answers is not in the document]
Comment (tarasSanotskyi) [reply src/components/login-dialog/LoginDialog.js]: fixed

File: src/components/login-form/LoginForm.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Box, FormControlLabel, Typography, TextField, Button, Checkbox,IconButton, InputAdornment } from '@mui/material'
+import Visibility from '@mui/icons-material/Visibility'
+import VisibilityOff from '@mui/icons-material/VisibilityOff'
+
+import style from './login-form.style'
+
+const LoginForm = ({ handleSubmit, handleChange, handleBlur, data, errors }) => {
```
[the remark this answers is not in the document]
Comment (tarasSanotskyi) [reply src/components/login-form/LoginForm.js]: fixed

File: src/containers/login-dialog/LoginDialog.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+import GoogleLogin from '~/containers/google-login/GoogleLogin'
+import LoginForm from '~/containers/login-form/LoginForm'
+import useForm from '~/hooks/use-form'
+import useConfirm from '~/hooks/use-confirm'
+import usePrompt from '~/hooks/use-prompt'
+import { email, password } from '~/constants/validation/login'
+import login from '~/assets/img/login-dialog/login.svg'
+import style from '~/containers/login-dialog/login-dialog.style'
+
+const LoginDialog = () => {
```
[the remark this answers is not in the document]
Comment (tarasSanotskyi) [reply src/containers/login-dialog/LoginDialog.js]: fixed

**Pull request merged**